### PR TITLE
hw09

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**
check please : )

### CODE REVIEW

File: src/main/java/homework09/ExtraMainFutureTask.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	public static void main(String[] args) {
+		ExecutorService es = Executors.newFixedThreadPool(2);
+		
+		Callable<String> callable1 = new CountThread(300);
+		Callable<String> callable2 = new CountThread(1000);
+
+		FutureTask<String> ft1 = new FutureTask<String>(callable1);
+		FutureTask<String> ft2 = new FutureTask<String>(callable2);
+
+		es.execute(ft1);
```
Comment (Cherkashyn) [inline src/main/java/homework09/ExtraMainFutureTask.java:22]: ExecutorService uses method submit() for a Callable implementations.

File: src/main/java/homework09/ExtraMainFutureTask.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.FutureTask;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class ExtraMainFutureTask {
+
+	public static void main(String[] args) {
+		ExecutorService es = Executors.newFixedThreadPool(2);
```
Comment (Cherkashyn) [inline src/main/java/homework09/ExtraMainFutureTask.java:14]: Variable names are very short :)

**Review by Cherkashyn — APPROVED** [review]
Ok. Read comments.

File: src/main/java/homework09/ExtraMainFutureTask.java (hunk 1 above)
Comment (u2107680007911543774e7r) [reply src/main/java/homework09/ExtraMainFutureTask.java:22]: > ExecutorService uses method submit() for a Callable implementations.

Yea, I know. I use submit with Future in the next file containing Main method, but this is another example not really mentioned in home task. It is FutureTask, I just did this Main for myself to learn : ) Maybe I shouldn't add this file to commit, my bad.

File: src/main/java/homework09/ExtraMainFutureTask.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+		Callable<String> callable1 = new CountThread(300);
+		Callable<String> callable2 = new CountThread(1000);
+
+		FutureTask<String> ft1 = new FutureTask<String>(callable1);
+		FutureTask<String> ft2 = new FutureTask<String>(callable2);
+
+		es.execute(ft1);
+		es.execute(ft2);
+
+		while (true) {
```
Comment (SerhiiPasko) [inline src/main/java/homework09/ExtraMainFutureTask.java:25]: How you normally can exit from there?

**Review by SerhiiPasko — APPROVED** [review]
Read my comment

**Pull request merged**